docs/policies/support.md

```diff
@@ -56,6 +56,6 @@ Azure SDK 라이브러리는 Azure Service REST API, 프로그래밍 언어 런
 - [JavaScript 테스트 구성](https://github.com/Azure/azure-sdk-for-js/blob/main/eng/pipelines/templates/stages/platform-matrix.json)
 - [Python 테스트 구성](https://github.com/Azure/azure-sdk-for-python/blob/main/eng/pipelines/templates/stages/platform-matrix.json)
 
-### **지원**:
+### **지원:**
```

Comment:
I am not sure whether including ":" to bold fonts is a good idea - or just removing ":" would be fine?